vsce package missing as a dependency

When you try to run the following command :

npm install && npm run build && npm run package

you will get an error like below 

```
> @zowe/cli@6.13.0 postinstall C:\Users\test2020\vscode-extension-for-zowe-api-sample\node_modules\@zowe\cli
> node ./scripts/validatePlugins

Since you re-installed Zowe CLI, we are re-validating any plugins.

_____ Validation results for plugin '@zowe/zos-ftp-for-zowe-cli' _____
This plugin was successfully validated. Enjoy the plugin.
added 148 packages from 148 contributors and audited 150 packages in 8.38s
found 0 vulnerabilities


> zowe-explorer-ftp-extension@1.5.1 build C:\Users\test2020\vscode-extension-for-zowe-api-sample
> tsc -p ./


> zowe-explorer-ftp-extension@1.5.1 package C:\Users\test2020\vscode-extension-for-zowe-api-sample
> vsce package

'vsce' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zowe-explorer-ftp-extension@1.5.1 package: `vsce package`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the zowe-explorer-ftp-extension@1.5.1 package script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\AppData\Roaming\npm-cache\_logs\2020-05-29T09_40_08_794Z-debug.log
```


This is because the vsce needs to be installed globally which can be done either via the following command 

npm install -g vsce or you can run it locally to fix this 
eg 
npm install vsce

--- a/package.json
+++ b/package.json
@@ -33,8 +33,9 @@
 	"dependencies": {
 		"@zowe/cli": "6.13.0",
 		"@zowe/zos-ftp-for-zowe-cli": "1.0.2",
-		"vscode-nls": "4.1.2",
-		"tmp": "0.2.0"
+		"tmp": "0.2.0",
+		"vsce": "^1.75.0",
+		"vscode-nls": "4.1.2"
 	},
 	"devDependencies": {
 		"@types/glob": "^7.1.1",

--- a/package.json
+++ b/package.json
@@ -33,9 +33,7 @@
 	"dependencies": {
 		"@zowe/cli": "6.13.0",
 		"@zowe/zos-ftp-for-zowe-cli": "1.0.2",
-		"tmp": "0.2.0",
-		"vsce": "^1.75.0",
-		"vscode-nls": "4.1.2"
+		"tmp": "0.2.0"
 	},
 	"devDependencies": {
 		"@types/glob": "^7.1.1",
@@ -44,6 +42,7 @@
 		"@types/tmp": "0.2.0",
 		"glob": "^7.1.6",
 		"typescript": "^3.8.3",
-		"tslint": "6.1.2"
+		"tslint": "6.1.2",
+		"vsce": "^1.75.0"
 	}
 }

